.markdownlint-cli2.jsonc

```diff
@@ -5,6 +5,9 @@
       "autolink": false,
       "shortcut": false
     },
+    "MD049": {
+      "style": "underscore"
+    },
```

Comment:
my markdownlint plugin was replacing all underscores with asterisks, so added this to specify that underscores are preferred for denoting emphasis